cmd/containerboot/main.go

```diff
@@ -208,6 +208,24 @@ func main() {
 		log.Fatalf("failed to watch tailscaled for updates: %v", err)
 	}
 
+	// Now that we've started tailscaled, we can symlink the socket to the
+	// default location if needed.
+	const defaultTailscaledSocketPath = "/var/run/tailscale/tailscaled.sock"
+	if cfg.Socket != "" && cfg.Socket != defaultTailscaledSocketPath {
+		// If we were given a socket path, symlink it to the default location so
+		// that the CLI can find it without any extra flags.
+		// See #6849.
+
+		dir := filepath.Dir(defaultTailscaledSocketPath)
+		if err := os.MkdirAll(dir, 0700); err == nil {
+			if err := syscall.Symlink(cfg.Socket, defaultTailscaledSocketPath); err != nil {
+				log.Printf("[warning] failed to symlink socket: %v", err)
```

Comment:
Nit: you could maybe print here that now they will have to use the --socket path if they want to run `tailscale`